### PR TITLE
BUG 1858400: [Performance] Lease refresh period for machine-api-controllers is too high, causes heavy writes to etcd at idle

### PR DESCRIPTION
**Pull request by Danil-Grigorev**
Reduce default lease retry rate on `30s` which will prevent from heavy writes into etcd at idle, and constrain renew deadline on `90s`.

Inspired by https://github.com/openshift/cloud-credential-operator/pull/231

### CODE REVIEW

**Comment by Danil-Grigorev** [comment]
/hold cancel
Set this on `60s`, so it will try once per renew deadline.

**Comment by Danil-Grigorev** [comment]
After some realization, set it on `30s` and `90s`, so it would retry at least one to twice.

**Comment by elmiko** [comment]
in general i think this is a good patch, but i agree with @michaelgugino that the values should be higher.

**Comment by michaelgugino** [comment]
Okay, after some more discussion, we determined 120/110/90 might be a better fit.

Don't want the leadership to be too long as if the pod gets moved (eg, upgrades) we don't want operations suspended for too long.  120 seconds would be adequate.

File: pkg/operator/sync.go [hunk 1]
```diff
@@ -446,7 +446,7 @@ func newContainers(config *OperatorConfig, features map[string]bool) []corev1.Co
 		"--logtostderr=true",
 		"--v=3",
 		"--leader-elect=true",
-		"--leader-elect-lease-duration=90s",
+		"--leader-elect-lease-duration=120s",
```
Comment (elmiko) [inline pkg/operator/sync.go:449]: this is the only confusing part to me. we set the lease duration in the controller-runtime config options, do we also need to set on the command line?

Comment (michaelgugino) [reply pkg/operator/sync.go:449]: We use the default if not specified on the CLI.  This is useful for development/debugging.

Comment (elmiko) [reply pkg/operator/sync.go:449]: ahh, perfect. thanks for the explanation Mike!

**Pull request merged**